cpp/collections_module/algorithms/collections.hpp

```diff
@@ -0,0 +1,23 @@
+#include <mgp.hpp>
+
+
+void unionAll(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory){
+  mgp::memory=memory;
+  const auto arguments=mgp::List(args);
```

Comment:
empty space between arugments and =, and = and mgp::List. I think we need to apply formatting here, ping me on slack to discuss about formatting